pkg/grapicmd/di/providers.go

```diff
@@ -23,70 +23,70 @@ var (
 	gexCfgMu sync.Mutex
 )
 
-func ProvideUI(cfg *grapicmd.Config) clui.UI {
+func ProvideUI(ctx *grapicmd.Ctx) clui.UI {
 	uiMu.Lock()
 	defer uiMu.Unlock()
 	if ui == nil {
-		ui = clui.New(cfg.OutWriter, cfg.InReader)
+		ui = clui.New(ctx.OutWriter, ctx.InReader)
 	}
 	return ui
 }
 
-func ProvideCommandExecutor(cfg *grapicmd.Config, ui clui.UI) command.Executor {
-	return command.NewExecutor(cfg.OutWriter, cfg.ErrWriter, cfg.InReader)
+func ProvideCommandExecutor(ctx *grapicmd.Ctx, ui clui.UI) command.Executor {
+	return command.NewExecutor(ctx.OutWriter, ctx.ErrWriter, ctx.InReader)
 }
 
-func ProvideGenerator(cfg *grapicmd.Config, ui clui.UI) module.Generator {
+func ProvideGenerator(ctx *grapicmd.Ctx, ui clui.UI) module.Generator {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideGenerator should have comment or be unexported